test/atm/global-workflow/jjob_ens_letkf.sh

```diff
@@ -53,7 +53,9 @@ cp $EXPDIR/config.base_lobsdiag_forenkf_false $EXPDIR/config.base
 
 # Execute j-job
 if [[ $machine = 'HERA' || $machine = 'ORION' || $machine = 'HERCULES' ]]; then
-    sbatch --nodes=1 --ntasks=36 --account=$ACCOUNT --qos=batch --time=00:30:00 --export=ALL --wait --output=atmensanlletkf-%j.out ${HOMEgfs}/jobs/JGLOBAL_ATMENS_ANALYSIS_LETKF
+    sbatch --nodes=1 --ntasks=6 --account=$ACCOUNT --qos=batch --time=00:30:00 --export=ALL --wait --output=atmensanlletkf-%j.out ${HOMEgfs}/jobs/JGLOBAL_ATMENS_ANALYSIS_LETKF
+elif [[ $machine = 'URSA' ]]; then
+     sbatch --nodes=1 --ntasks=6 --account=$ACCOUNT --qos=batch --partition=u1-compute --mem="96Gb" --time=00:30:00 --export=ALL --wait --output=atmensanlletkf-%j.out ${HOMEgfs}/jobs/JGLOBAL_ATMENS_ANALYSIS_LETKF
```

Comment:
It's interesting that so much more memory is required on Ursa than the other machines.  This seems to be a known issue with spack-stack 1.9.1 as documented in this issue: https://github.com/JCSDA/spack-stack/issues/1588 and something that will need to be accounted for when moving other systems to 1.9.1.